core/src/main/java/com/datastax/oss/driver/api/core/config/DriverConfigLoader.java

```diff
@@ -211,21 +211,27 @@ static ProgrammaticDriverConfigLoaderBuilder programmaticBuilder() {
   void onDriverInit(@NonNull DriverContext context);
 
   /**
-   * Triggers an immediate reload attempt.
-   *
-   * @return a stage that completes once the attempt is finished, with a boolean indicating whether
-   *     the configuration changed as a result of this reload. If so, it's also guaranteed that
-   *     internal driver components have been notified by that time; note however that some react to
-   *     the notification asynchronously, so they may not have completely applied all resulting
-   *     changes yet. If this loader does not support programmatic reloading &mdash; which you can
-   *     check by calling {@link #supportsReloading()} before this method &mdash; the returned
-   *     object will fail immediately with an {@link UnsupportedOperationException}.
+   * Triggers an immediate reload attempt and returns a stage that completes once the attempt is
+   * finished, with a boolean indicating whether the configuration changed as a result of this
+   * reload.
+   *
+   * <p>If so, it's also guaranteed that internal driver components have been notified by that time;
+   * note however that some react to the notification asynchronously, so they may not have
+   * completely applied all resulting changes yet.
+   *
+   * <p>If this loader does not support programmatic reloading &mdash; which you can check by
+   * calling {@link #supportsReloading()} before this method &mdash; the returned stage should fail
+   * immediately with an {@link UnsupportedOperationException}. The default implementation of this
+   * interface does support programmatic reloading however, and never returns a failed stage.
```

Comment:
Mandatory rant about off-topic cosmetic changes: the javadocs might look marginally better with paragraphs, but this produces a large diff that buries the actual change.

Reply:
Which is why the changes were confined to a separate commit, see bed37a7200356fa266be20c3cd8cb470e1fec76f.

Reply:
OK I missed that 👍